pkg/auth/webapp/redirect.go

```diff
@@ -7,6 +7,10 @@ import (
 	"github.com/authgear/authgear-server/pkg/util/httputil"
 )
 
+var reservedRedirectURIs = []string{
+	"authgearsdk://host/path", // For Authgear SDK only, used for closing the webview
+}
```

Comment:
I think we can simply allow all uri with scheme `authgearsdk://`?

Reply:
Wanted to be specific since this is the only use case, updated to compare schema instead 🙏